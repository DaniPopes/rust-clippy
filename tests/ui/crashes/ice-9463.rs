fn main() {
    let _x = -1_i32 >> -1;
    let _y = 1u32 >> 10000000000000u32;
    //~^ ERROR: literal out of range for `u32`
    //~| NOTE: the literal `10000000000000u32` does not fit into the type `u32` whose range is
}
