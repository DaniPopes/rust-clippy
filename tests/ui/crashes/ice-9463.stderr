error: literal out of range for `u32`
  --> tests/ui/crashes/ice-9463.rs:3:22
   |
LL |     let _y = 1u32 >> 10000000000000u32;
   |                      ^^^^^^^^^^^^^^^^^
   |
   = note: the literal `10000000000000u32` does not fit into the type `u32` whose range is `0..=4294967295`
   = note: `#[deny(overflowing_literals)]` on by default

error: aborting due to 1 previous error

